www/js/group/createGroup/createGroup.js

```diff
@@ -1,31 +1,11 @@
 'use strict';
 
-angular.module('main').controller('CreateGroupCtrl', ['$rootScope', '$scope', '$state', '$log', 'ionicTimePicker', 'GroupFactory', function ($rootScope, $scope, $state, $log, ionicTimePicker, GroupFactory) {
-
-  var ipObj1 = {
-    callback: function (val) {      //Mandatory
-      if (typeof (val) === 'undefined') {
-        console.log('Time not selected');
-      } else {
-        var selectedTime = new Date(val * 1000);
-        console.log('Selected epoch is : ', val, 'and the time is ', selectedTime.getUTCHours(), 'H :', selectedTime.getUTCMinutes(), 'M');
-        $scope.group.expirationTime = selectedTime;
-        // GroupFactory.createGroup(groupInfo)
-        // .then(() => $state.go('camera'))
-        // .catch($log.error);
-      }
-    },
-    inputTime: 50400,   //Optional
-    format: 12,         //Optional
-    step: 15,           //Optional
-    setLabel: 'Set2'    //Optional
-  };
+angular.module('main').controller('CreateGroupCtrl', ['$scope', '$state', 'GroupFactory', function ( $scope, $state, GroupFactory) {
 
   $scope.createGroup = function(groupDetails) {
-    // ionicTimePicker.openTimePicker(ipObj1);
     GroupFactory.createGroup(groupDetails)
     .then(function(){
-      $state.go('tab.group-logged-in');
+      $state.go('tab.groups');
     })
     .catch(function(err){
       console.log(err);
```

Comment:
You might check out something like `toast` or `growl`, libraries that can place a temporary message on the ui. It's a good cheap way to report errors back to the end user. [Here's one](https://github.com/rajeshwarpatlolla/ionic-toast) somebody built specifically for ionic—there are others out there too.